app-admin/tpm/tpm-1.3.3.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Tiny password manager"
+HOMEPAGE="https://github.com/nmeum/tpm"
+if [[ "${PV}" == *9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/nmeum/${PN}.git"
+else
+	SRC_URI="https://github.com/nmeum/${PN}/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-3"
+SLOT="0"
+
+RDEPEND="app-crypt/gnupg"
+BDEPEND="dev-lang/perl"
+
+src_compile() {
+	emake
+}
+
+src_install() {
+	emake install DESTDIR="${ED}" PREFIX="${EPREFIX}/usr"
```

Comment:
Does it work without this? It might not if they're doing prefix in a weird way.

If we do need to keep this, we should only account for prefix (in the Gentoo sense) once: either do DESTDIR="${D}" or PREFIX="${EPREFIX}/usr", but not both (because ${ED}=${D}/${EPREFIX}). I'd change DESTDIR to be ${D}.

Reply:
It needs `PREFIX` as a `make install` argument, but thanks for explaining about `${ED}` vs `${D}`. I changed it to `DESTDIR=${D}`.